test/cpp/qps/client.h

```diff
@@ -54,6 +54,10 @@
 #include "test/cpp/qps/usage_timer.h"
 #include "test/cpp/util/create_test_channel.h"
 
+extern "C" {
+#include "src/core/lib/surface/completion_queue.h"
```

Comment:
Oh, I think my last comment was not clear. I meant that this extern "C" business should go inside `completion_queue.h` itself (within an `#ifdef __cplusplus` like the way it is used in many other files.)